docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.lists.scss

```diff
@@ -98,6 +114,14 @@
     @include hb-ext-icon-color('black');
     background-image: none;
 
+    
```

Comment:
Let's remove this extra line. It is throwing an error when you run `npm run test`.

![failing test](https://user-images.githubusercontent.com/12678977/103790444-35aa0b00-500f-11eb-9222-e01f38c259e3.png)

Reply:
Good catch, fixed up.